code-challenges.js

```diff
@@ -9,31 +9,95 @@
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 42
-// var temp = 350
-// var temp = 212
+var temp1 = 42
+var temp2 = 350
+var temp3 = 212
+
+var temp = [temp1, temp2, temp3]//Created an array for the three variables for temperature
+
+
+//Create a iteration of the array and output a statement based on the temp[index] value.
+for (let i = 0; i < temp.length; i++) {
+
+    if (temp[i] > 212) {
+        console.log(`${temp[i]} is above boiling point`)
+    }
+    else if (temp[i] < 212) {
+        console.log(`${temp[i]} is below boiling point`)
+    }
+    else { console.log(`${temp[i]} is at boiling point`) }
+}
+
+
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 42, 9, 13]
 
+var newNumSet = [] //Create an empty new array
+
+
+//Iterate to copy values of myNumbers1 to the new array
+for (let i = 0; i < myNumbers1.length; i++) {
+    newNumSet.push(myNumbers1[i]);
+}
+
+//Iterate to copy values of myNumbers2 to the new array
+for (let i = 0; i < myNumbers2.length; i++) {
+    newNumSet.push(myNumbers2[i]);
+}
+
+
+console.log(newNumSet.length); //Output the length of the new array
+
```

Comment:
The final output looks good, but there is a quicker way to get to the end goal using built in methods.